### ISSUE TITLE
Why does gfRepeater_getChoiceValue return a label instead of a value?

### ISSUE THREAD

**Issue by stuporglue** [kodie/gravityforms-repeater#22]
Seems like this code: 

```
var id = this.id;
var label = jQuery(this).siblings('label').first().text();
value.push(label);
```

Should just be: 

```
value.push(jQuery(this).val());
```

The value and label will typically be the same, unless the user checks the "show values" checkbox. I believe that checking the value is the correct thing to do and should always work.

![should_be_value_maybe](https://cloud.githubusercontent.com/assets/16576/12933138/38048770-cf4c-11e5-9be4-9a9e4800ad7a.png)

**Comment by kodie** [comment]
Good idea! I will fix this in the next release!

**Comment by kodie** [comment]
Fixed in cb191f1 commit in latest version! Thanks again!
